taskqueue/taskqueue.go

```diff
@@ -147,6 +148,47 @@ func NewPOSTTask(path string, params url.Values) *Task {
 	}
 }
 
+// RequestHeaders are the special HTTP request headers available to push task
+// HTTP request handlers. These headers are set internally by App Engine.
+// See https://cloud.google.com/appengine/docs/standard/go/taskqueue/push/creating-handlers#reading_request_headers
+// for a description of the fields.
+type RequestHeaders struct {
+	QueueName          string
+	TaskName           string
+	TaskRetryCount     int64
+	TaskExecutionCount int64
+	TaskETA            time.Time
+
+	TaskPreviousResponse int
+	TaskRetryReason      string
+	FailFast             bool
+}
+
+// ParseRequestHeaders parses the special HTTP request headers available to push
+// task request handlers.
```

Comment:
a comment that it drops header values of the wrong format silently?
i think this would be helpful for future debugging / api changes / etc.